.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -1,26 +1,34 @@
 ---
 name: Onboarding
 about: An onboarding checklist for folks joining the team!
-title: Onboarding checklist for {Github Username}
+title: Onboarding checklist for {GitHub Username}
 labels: ''
 assignees: ''
 
 ---
-
-### Welcome to the team  {GH USERNAME}!
+### Welcome to the team @{GitHub Username}!
 
 Here's a few pointers to get you started as well as a checklist to make sure you've got access to :all-the-things:!
 
 **For you**
-- [ ] [Join](https://github.com/GSA/GitHub-Administration#joining-the-gsa-enterprise-organization) GSA’s Enterprise Github org
 - [ ] Review the [project README](https://docs.google.com/document/d/1g8nYqYS_ifFlZB-DBgfeSoJRMB__EqWsmLnacyk-bDI/) in drive
-- [ ] Join Slack channels in the project README
+- [ ] Join the Slack channels in the project README
+- [ ] Review our [process documentation](https://github.com/GSA-TTS/FAC#fac-documentation)
+- [ ] Get access to our GitHub repositories
+  - [ ] If you are NOT already a member of the [GSA](https://github.com/orgs/GSA/people) or [GSA-TTS](https://github.com/orgs/GSA-TTS/people) GitHub organizations, [follow the process outlined here](https://github.com/GSA/GitHub-Administration#joining-the-gsa-enterprise-organization), requesting access to the appropriate org(s). Include the Product Lead (see [the staffing list](https://docs.google.com/document/d/1g8nYqYS_ifFlZB-DBgfeSoJRMB__EqWsmLnacyk-bDI/edit#heading=h.us8xylqg455c)) on cc: for your email, and mention that you’re being added to the Federal Audit Clearinghouse project. (This may reduce back-and-forth as you’re being added to the GitHub orgs.) Note this step potentially takes a few days; there’s a person who does things in order for this to happen.
+  - [ ] Once you are added to the GSA-TTS org, ask the person(s) with the "Maintainer" role to add you to [the FAC-team in GitHub](https://github.com/orgs/GSA-TTS/teams/fac-team/members). This will give you read/write access to our repositories.
```

Comment:
Maintainers of a repo aren't always easy to find on GitHub. Should we just list who they are or add a [CODEOWNERS file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) to the repo and link that?

Reply:
and/or move this as an onboarding buddy issue?